CHANGELOG

```diff
@@ -1,6 +1,8 @@
 v3.5.1 (XXXX-XX-XX)
 -------------------
 
+* Drop collection action with shorter timeout
```

Comment:
As in 3.4.8 (feel free to propose Manhatten style english as long as it is consistent with the overall changelog format).